WeakAuras/GenericTrigger.lua

```diff
@@ -2298,6 +2305,7 @@ do
     cdReadyFrame:RegisterEvent("SPELL_UPDATE_COOLDOWN");
     cdReadyFrame:RegisterEvent("UNIT_SPELLCAST_SENT");
     cdReadyFrame:RegisterEvent("BAG_UPDATE_DELAYED");
+    cdReadyFrame:RegisterEvent("BAG_UPDATE_COOLDOWN");
```

Comment:
The CD tracking of health stones and trinkets seems to work since they fire  SPELL_UPDATE_COOLDOWN. So unless you know of a item whose item cooldown tracking doesn't work, I wouldn't touch the cdReadyFrame thing.